website/docs/gettingstarted/safety-guidelines.md

```diff
@@ -0,0 +1,124 @@
+---
+sidebar_position: 2
+---
+
+# Safety guidelines
+
+:::caution
+Please read this guide carefully before operating OpenArm and use it safely at your own risk.
+:::
+
+We aim to realize a society where robots can be closer to people’s lives and provide meaningful support in daily activities.
+To achieve this, it is essential to ensure the safety and peace of mind of everyone involved in the development of robots, the environments where robots operate, and the people who interact with them.
+This guideline outlines key points you should be aware of to use OpenArm safely and responsibly.
+Please understand that the information presented here is only an example; true safety can only be ensured through sincere risk assessment and continuous improvement of safety measures.
+We hope you will become a member of the OpenArm community and join us in upholding and expanding this culture of safety.
+
+
+## Safe Use Requirements
+
+### 1. Install in a safe location
+
+<div align="center">
+  <img src="/img/gettingstarted/safety-guidelines/sample.png" alt="sample" width="700" />
+</div>
```

Comment:
How about using the `BlockImage` component or defining the component for gettingstarted like `AssemblyGuideImage`?
ref: https://github.com/enactic/openarm/blob/main/website/src/components/AssemblyGuideImage.tsx